frappe/desk/doctype/event/event.py

```diff
@@ -30,6 +30,12 @@ def validate(self):
 		if getdate(self.starts_on) == getdate(self.ends_on) and self.repeat_on == "Every Day":
 			frappe.msgprint(frappe._("Every day events should finish on the same day."), raise_exception=True)
 
+	def after_insert(self):
```

Comment:
you can also check `no_copy` on this id!

Reply:
Humm yes I didn't think about that. Let me see how I can modifiy the creation of the custom field in the data migration tool!